package.json

```diff
@@ -34,6 +34,7 @@
     "@livekit/changesets-changelog-github": "^0.0.4",
     "@types/node": "^20.10.1",
     "@typescript-eslint/eslint-plugin": "^4.29.1",
+    "@types/node-jose": "^1.1.13",
```

Comment:
`jose` ships already with types ootb. Adding another types package explicitly might just lead to some inaccuracies there. What's the intention behind adding this? (looks like the types are actually for cisco's fork of the library)

Reply:
ah I'll revert this. the types didn't show up for me in VS code.